tests/event-bus/Gopkg.toml

```diff
@@ -2,18 +2,29 @@ required = [
   "golang.org/x/lint/golint",
   "golang.org/x/tools/cmd/goimports",
 ]
-
+# TODO Fix me please
```

Comment:
As your todo already indicates, we need to wait until "knative-eventing-compo-0.10.0" is merged to master